src/components/App/index.spec.tsx

```diff
@@ -23,22 +25,15 @@ describe(__filename, () => {
   }: RenderParams = {}) => {
     const contextWithRouter = createContextWithFakeRouter();
     const context = {
-      ...contextWithRouter,
```

Comment:
@willdurand please take a look at this. I didn't understand why you were creating an object called `context` that only contains a single property called `context`, so I removed that. It could be that there might be something other than the `context` property in the object returned by `createContextWithFakeRouter()` at some point, but for now there is not, so the extra wrapping didn't really make sense to me.

Reply:
`createContextWithFakeRouter()` is supposed to return:

```
{
  context: {},
  childContextTypes: {}
}
```

Reply:
Thanks @willdurand, I've put it back the way it was.